schemas/reports/report.yaml

```diff
@@ -20,6 +20,7 @@ oneOf:
   - $ref: us_driving_licence_report.yaml
   - $ref: device_intelligence_report.yaml
   - $ref: india_pan_report.yaml
+  - $ref: generic_report.yaml
```

Comment:
This schema has been created to have PHP generator not natively supporting `oneOf` yet to present a generic report schema, otherwise generated code for report class would have been looked weird as including fields from last report type in OpenAPI definition ([india_pan_report](url))

Reply:
Putting this change aside for the time being. Code is looking a little bit weird due to the presence of `get/setProperties` and `get/setBreakdown` methods with `IndiaPanReport*` types in Report object, but at least we don't impact other generators.